readme.md

```diff
@@ -133,35 +133,41 @@ You can run the Wagtail demo locally without setting up Vagrant or Docker and si
 
 With [PIP](https://github.com/pypa/pip) and [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
 installed, run:
-
+```bash
     mkvirtualenv wagtailbakerydemo
```

Comment:
Please remove the indenting from these blocks - the backtick notation eliminates the need for it, so leaving it in place means that they're indented too much.

Reply:
Fixed this!